index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header> 
+         <p>Jhon Neira</p>
+         <div class="menu-icon">
+            <img src="Images/Menu.png" alt="menu-icon">
+         </div> 
+    </header>
+    <main>
+        <h1>
+            Hey There. <br>
+            I'm Alejandro
+        </h1>
+        <h2>
+            I am a Software Developer
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don't hestiate to contact me.
+        </p>
+        <div class="Social-container">
+            <img class="Gato" src="Images/Gato.png" alt="GitHub">
```

Comment:
- [ ] Kindly use the `<ul>` tags and `<li>` tags when you are going to list multiple elements following the good semantics for HTML,[ please refer to this article ](https://www.w3schools.com/tags/tag_ul.asp)